src/lms-harmonizer/tests_integration_sql/migrator_helper_mssql.py

```diff
@@ -0,0 +1,161 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+from sqlalchemy.engine.base import Connection
+
+# This is a copy/paste shortcut to initialize the test database with the proper structure
+
+
+def migrate_lms_user_and_edfi_student(connection: Connection):
+    connection.execute(
+        """
+IF NOT EXISTS (SELECT * FROM sys.schemas WHERE name = N'edfi')
+EXEC sys.sp_executesql N'CREATE SCHEMA edfi';
+        """
+    )
+
+    connection.execute(
+        """
+IF NOT EXISTS (SELECT * FROM sys.schemas WHERE name = N'lms')
+EXEC sys.sp_executesql N'CREATE SCHEMA lms';
+        """
+    )
+
+    # from edfi_lms_ds_loader's create_user_tables.sql and add_mapping_columns_for_edfi_student_and_section.sql
+    connection.execute(
+        """
```

Comment:
I like the idea, but since these scripts for creating the tables are not relevant themselves (I mean, won't change too much and is not needed to understand the test itself), I'd put the scripts in a separate class or something like that to improve the readability of the methods 🤔

Reply:
I like that idea, so that we can just read something like

```python
    connection.execute(CREATE_EDFI_SCHEMA)
    connection.execute(CREATE_LMS_SCHEMA)
    connection.execute(CREATE_LMS_TABLES)
    # etc.
```

Reply:
Done